inverse_covariance/tests/clean_test.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+from scipy import sparse
+
+from sklearn.utils.testing import assert_raises
+
+from inverse_covariance.clean import (
+   twoway_standardize
+)
+
```

Comment:
Can we add some tests for the `TwoWayStandardScaler` with fixed inputs and expected outputs that run over a number of options and ensure it's right.  Also the standard sklearn check_estimator test (see example here: https://github.com/skggm/skggm/blob/develop/inverse_covariance/tests/common_test.py)